ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouterMetrics.java

```diff
@@ -31,42 +37,139 @@
   // @todo: Ensure all metrics here get updated appropriately.
   // @todo: chunk filling rate metrics.
   // @todo: More metrics for the RequestResponse handling (poll, handleResponse etc.)
-  // Rates
+
+  // Operation rate.
   public final Meter putBlobOperationRate;
+  public final Meter getBlobInfoOperationRate;
+  public final Meter getBlobOperationRate;
+  public final Meter deleteBlobOperationRate;
   public final Meter operationQueuingRate;
   public final Meter operationDequeuingRate;
 
+  // Latency.
   public final Histogram putBlobOperationLatencyMs;
   public final Histogram putChunkOperationLatencyMs;
+  public final Histogram getBlobInfoOperationLatencyMs;
+  public final Histogram getBlobOperationLatencyMs;
+  public final Histogram deleteBlobOperationLatencyMs;
   public final Histogram routerRequestLatencyMs;
```

Comment:
those are at operation level. the purpose of this metric is to track the latency for each request sent out by the router.